storage/database/sharded_database_test.go

```diff
@@ -0,0 +1,271 @@
+package database
+
+import (
+	"bytes"
+	"context"
+	"io/ioutil"
+	"os"
+	"reflect"
+	"sort"
+	"sync"
+	"testing"
+
+	"github.com/klaytn/klaytn/common"
+	"github.com/stretchr/testify/assert"
+)
+
+var ShardedDBConfig = []*DBConfig{
+	{DBType: LevelDB, SingleDB: false, NumStateTrieShards: 4, ParallelDBWrite: true},
+	{DBType: LevelDB, SingleDB: false, NumStateTrieShards: 8, ParallelDBWrite: true},
+	{DBType: LevelDB, SingleDB: false, NumStateTrieShards: 16, ParallelDBWrite: true},
+}
+
+func createEntries(entryNum int) []entry {
+	entries := make([]entry, entryNum)
+	for i := 0; i < entryNum; i++ {
+		//entries[i] = entry{common.MakeRandomBytes(10), common.MakeRandomBytes(10)}
+		entries[i] = entry{common.MakeRandomBytes(256), common.MakeRandomBytes(600)}
+	}
+	return entries
+}
+
+// testIterator tests if given iterator iterates all entries
+func testIterator(t *testing.T, checkOrder bool, entriesFromIterator func(db shardedDB, entryNum int) []entry) {
+	entryNum := 500
```

Comment:
Could you test several entryNum case like below?
- less than NumStateTrieShards
- same with NumStateTrieShards
- upper than NumStateTrieShards
```suggestion
	entryNum := 500
```

Reply:
I changed `testIterator`. entryNum case is now
* shardedDBCombineChanSize + 10 
* NumStateTrieShards - 1
* NumStateTrieShards
* NumStateTrieShards + 1